beacon.go

```diff
@@ -90,21 +90,20 @@ func query(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func genomeExists(ctx context.Context, refName string, allele string, coord int64) (bool, error) {
+func genomeExists(ctx context.Context, params queryParams) (bool, error) {
+	var w where
```

Comment:
Not sure you really need the type and method.

At first glance this looks like

where := strings.Join([]string{
  ...
}, " AND ")

to me... will that not work?

Reply:
I was trying to avoid empty clauses without a lot of duplicate code. I refactored to use strings.join